EpiserverRedirects.EntityFramework/Repository/QueryExtension.cs

```diff
@@ -0,0 +1,177 @@
+﻿using EPiServer.Shell.Services.Rest;
+using Forte.EpiserverRedirects.DynamicDataStore;
+using Forte.EpiserverRedirects.EntityFramework.Model;
+using Forte.EpiserverRedirects.Model.RedirectRule;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+
+namespace Forte.EpiserverRedirects.EntityFramework.Repository
+{
+    public static class QueryExtension
+    {
+        public static IEnumerable<RedirectRuleEntity> ApplyQuery(this IQueryable<RedirectRuleEntity> redirectRules,
```

Comment:
Similarly, we should try to extract common code from this method

Reply:
The code in both QueryExtentions is specific to the entity class and repository implementation. It is not common. It is similar, as we use LINQ and same property names in both Repository implementations: DDS and EF.
I don't see any way to extract it.

Reply:
As I wrote in another comment - extracting a common interface could help here as well